File: XWSMicroservices/api_gateway/src/main/java/com/dislinkt/apigateway/dto/CommentDTO.java
package com.dislinkt.apigateway.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;

@AllArgsConstructor
@NoArgsConstructor
@Data
public class CommentDTO {
    public long postId;
    public long userId;
    public String content;
    public String dateCreated;
}

File: dislinkt-app-frontend/src/app/homepage/comment.dto.ts

export class CommentDTO{

    postId:number;
    username:String;
    userId:number;
    content:String;
    dateCreated:Date;
    constructor(){
        this.postId = 0;
        this.username="";
        this.userId = 0;
        this.content = "";
        this.dateCreated = new Date()
    }
}